Stripe/NSURLComponents+Stripe.m

```diff
@@ -0,0 +1,51 @@
+//
+//  NSURLComponents+Stripe.m
+//  Stripe
+//
+//  Created by Brian Dorfman on 1/26/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "NSURLComponents+Stripe.h"
+
+@implementation NSURLComponents (Stripe)
+
+- (NSDictionary<NSString *, NSString *> *)stp_queryItemsDictionary {
```

Comment:
Ah yep. Just forgot to resurrect those files when I copied this out of my old branch. I'll pull and update them.